package org.example.exceptions

class AuthErrorException : Exception()
